app/models/service_provider.rb

```diff
@@ -47,6 +47,14 @@ def encryption_opts
     }
   end
 
+  def skip_encryption_allowed
+    config = AppConfig.env.skip_encryption_allowed_list
+    return false unless config
```

Comment:
probably want to do `config.present?` because `JSON.parse` on empty string is bad times

```suggestion
    return false if config.blank?
```

I also dislike unless so trying with `if`